### PR TITLE
add a constitution

### PR DESCRIPTION
**Pull request by herdingdata**

### CODE REVIEW

File: constitution.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Membership
+
+A member is deemed as someone who is a member of our meetup group. 
+Membership is open for people to join and leave at any time. https://www.meetup.com/qcscot/members/
+
+## Subscriptions
+
+There is no cost of membership. On occasion we may decide to charge an 
+entry fee if we have to incur costs in order to put on an event, however we will endeavour to avoid
```
Comment (KakeLP) [inline constitution.md]: I suggest changing “however” to “but” to avoid comma-splicing.

File: constitution.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+## Subscriptions
+
+There is no cost of membership. On occasion we may decide to charge an 
+entry fee if we have to incur costs in order to put on an event, however we will endeavour to avoid 
+this wherever possible. 
+
+## Committee
+
+The committee exists primarily for the management of our finances. Other day to day activities are
+organised by the volunteers on #organisation-scotland slack channel on the queer code slack.
```
Comment (KakeLP) [inline constitution.md]: Suggest changing “volunteers on” to “volunteers on the”, “slack” to “Slack” (also in the line below), and “queer code slack” to “Queer Code Slack”.

File: constitution.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+The committee exists primarily for the management of our finances. Other day to day activities are
+organised by the volunteers on #organisation-scotland slack channel on the queer code slack. 
+See https://queer-code.org for details on how to join this slack.
+
+The committee will be formed of a minimum of two people: a chairperson and treasurer. 
+Additional committee members may be appointed as required.
+
+Appointment to the committee will be by agreement of the existing committee members.
+
+Any committee member may step down from their position by informing the remainder of the committee.
```
Comment (KakeLP) [inline constitution.md:44]: There seem to be no details on what happens if the size of the committee falls below two.

Comment (herdingdata) [reply constitution.md:44]: now added :)

**Review by zahara — COMMENTED** [review]
Maybe let's not tie ourselves to Meetup.com in this document?  Or at least say it's currently how we determine members, but may cover mailing lists etc and other means?

**Comment by herdingdata** [comment]
@zahara I've tweaked the wording to leave it a bit more open to future changes. Does that look ok?

**Review by zahara — APPROVED** [review]

**Pull request merged**